lib/validation.js

```diff
@@ -18,10 +18,12 @@
  *
  * @module numberValidation
  */
+const isDecimal = require('validator/lib/isDecimal');
+const isInt = require('validator/lib/isInt');
 
-module.exports = function numberValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
+module.exports = function numberValidation(input) {
+  if (input.target.value && !isInt(input.target.value.toString()) && !isDecimal(input.target.value.toString())) {
```

Comment:
Really? Character count? If we're going to play that, why isn't value stored as a var and referenced?

I understand what's up. My logic brain clearly isn't functioning today.